config/alpha.json

```diff
@@ -13,10 +13,11 @@
   "leoUrlRoot": "https://leonardo.dsde-alpha.broadinstitute.org",
   "marthaUrlRoot": "https://us-central1-broad-dsde-alpha.cloudfunctions.net",
   "orchestrationUrlRoot": "https://firecloud-orchestration.dsde-alpha.broadinstitute.org",
+  "profileUrlRoot": "https://profile-dot-broad-shibboleth-prod.appspot.com/dev",
```

Comment:
I thought about this as well. Although the new service currently only handles eRA Commons tokens, it's built and named ('terra-profile') in a way that suggests it could evolve into a general purpose profile service. It's unclear whether that will come to pass, but for now the name seems relatively unambiguous to me (Sam is limited to IAM, rather than profiles. Orch currently handles most of the profile stuff, but also does a lot of other things)